### PR TITLE
rosetta code : rock_paper_scissors

### PR DESCRIPTION
**Pull request by ventilooo**
Signed-off-by: Istvan SZALAI <szalai972@gmail.com>

### CODE REVIEW

**Comment by ventilooo** [comment]
rerol @privat

**Comment by ventilooo** [comment]
nitcoin @Morriar

File: examples/rosettacode/rock_paper_scissors.nit [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+var player_input = ""
+var tab = ["rock", "scissors", "paper"]
+var tab_message = ["You lose", "You win!", "It's a tie!"]
+loop
+	loop
+		print "Choose your weapon|exit"
+		player_input = gets
+		if player_input == "exit" then exit 0
+		if tab.has(player_input) then break
+		if not tab.has(player_input) then print "Invalid choice (rock/paper/scissors)"
```
Comment (Tagachi) [inline examples/rosettacode/rock_paper_scissors.nit]: You can just use ```else```

File: examples/rosettacode/rock_paper_scissors.nit [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+	if stdin.eof then return
```
Comment (Tagachi) [inline examples/rosettacode/rock_paper_scissors.nit]: Add a tabulation before ```if```

File: examples/rosettacode/rock_paper_scissors.nit [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+	if player == "rock" and computer == "scissors" then return 1
+	if player == "rock" and computer == "paper" then return 0
+	if player == "paper" and computer == "rock" then return 1
+	if player == "paper" and computer == "scissors" then return 0
+	if player == "scissors" and computer == "paper" then return 1
+	return 0
+end
+
+# Game loop.
+var index = 0
```
Comment (Tagachi) [inline examples/rosettacode/rock_paper_scissors.nit]: Useless variable

**Comment by R4PaSs** [comment]
Ping @ventilooo ? If you fix the issues mentioned by @Tagachi, this PR might result in a merge for my part, I can't see much else to work on right now

**Comment by privat** [comment]
ping?

**Comment by privat** [comment]
you might reopen if you update the PR

**Pull request closed without merging**